Source/WebKit/UIProcess/ProcessAssertion.h

```diff
@@ -116,6 +116,7 @@ class ProcessAssertion : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<
     Function<void()> m_prepareForInvalidationHandler;
     Function<void()> m_invalidationHandler;
 #if USE(EXTENSIONKIT)
+    static Lock s_capabilityLock;
```

Comment:
Why does the lock need to be static?

Reply:
And being static, is it even thread-safe?